public/client/components/GamesListItem.js

```diff
@@ -25,7 +25,7 @@ const GamesListItem = (props) => (
       {/* Key Features is bold and font color is white */}
       Key Features
       <ul>
-        {props.game.key_features.map(feature => <li>{feature}</li> )}
+        {props.game.key_features.map((feature, idx) => <li key={idx}>{feature}</li> )}
```

Comment:
I believe they recommend against using the index directly as a key. I usually add an identifier. No need to update for this PR, but something to keep in mind going forward.